packages/cdk/lib/cloudwatch-logs-management.ts

```diff
@@ -73,7 +73,7 @@ export class CloudwatchLogsManagement extends GuStack {
 		const setRetentionLambda = new GuScheduledLambda(this, 'set-retention', {
 			app: 'set-retention',
 			runtime: Runtime.NODEJS_16_X,
-			fileName: 'cloudwatch-logs-management.zip',
+			fileName: 'set-retention.zip',
```

Comment:
[@guardian/node-riffraff-artifact](https://github.com/guardian/node-riffraff-artifact) produces a zip file named `action/action.zip`. This is reasonable, so let's change.